mava/wrappers/README.md

```diff
@@ -10,15 +10,14 @@ instance which implements the [DeepMind Environment API][dm_env]. Mava currently
 * [PettingZoo][pettingzoo]
 * [SMAC][smac]
 * [Flatland][flatland]
-* [2D RoboCup][robocup]
 
 |<img  src="../../docs/images/multiw_animation.gif" width="400px"/> | <img src="../../docs/images/sc2_animation.gif" width="250px"/>
 |:---:|:---:|
 |MAD4PG on PettingZoo's Multi-Walker environment. | VDN on the SMAC 3m map.|
 
-|<img  src="../../docs/images/robocup_animation.gif" width="300px"/> | <img src="../../docs/images/flatland.gif" width="300px" />  |
-|:---:|:---:|
-|MAD4PG on the 2D RoboCup environment using 6 executors.| MADQN on Flatland. |
+| <img src="../../docs/images/flatland.gif" width="300px" />  |
+|:---:|
+|MADQN on Flatland. |
```

Comment:
If you can put all three pictures next to each other.

Reply:
Updated :+1: